backstop_data/engine_scripts/puppet/clickAndHoverHelper.js

```diff
@@ -27,7 +27,9 @@ module.exports = async (page, scenario) => {
   }
 
   if (postInteractionWait) {
-    await new Promise(resolve => setTimeout(resolve, postInteractionWait));
+    await new Promise(resolve => {
+      setTimeout(resolve, postInteractionWait);
```

Comment:
Ensure that `postInteractionWait` is a positive integer to avoid unexpected behavior in the `setTimeout` function.